src/foam/nanos/controller/AppStyles.js

```diff
@@ -255,5 +262,31 @@ foam.CLASS({
       scroll-behavior: auto !important;
     }
   }
+
+
+  .large-subtitle {
```

Comment:
for this one i just needed a bigger lineheight

Reply:
We should use the closest standard size type in that case and then just add one line height prop because here we have to specify the font-size again otherwise